.github/hfest_2022_resource.md

```diff
@@ -0,0 +1,36 @@
+[<img align="left" src="https://unskript.com/assets/favicon.png" width="100" height="100" style="padding-right: 5px">](https://unskript.com/assets/favicon.png) 
+<h1>Hacktoberfest 2022 Resource</h1>
+
+### Storage Buckets available for test
```

Comment:
Top headline can be
** Google cloud resources available for test**

Sub header can be Buckets (Object Storage), Compute instances...

Reply:
Took care of PR Comments.